test/sqllogictest/privilege_checks.slt

```diff
@@ -1779,68 +1777,35 @@ GRANT USAGE ON SCHEMA materialize.public TO joe;
 ----
 COMPLETE 0
 
-simple conn=joe,user=joe
-EXPLAIN SELECT * FROM v;
-----
-db error: ERROR: permission denied for VIEW "materialize.public.v"
-
-simple conn=child,user=child
+simple multiline,conn=joe,user=joe
 EXPLAIN SELECT * FROM v;
 ----
-db error: ERROR: permission denied for VIEW "materialize.public.v"
+Explained Query:
+  ReadStorage materialize.public.t
 
-simple conn=mz_system,user=mz_system
-GRANT SELECT ON TABLE v TO joe;
-----
-COMPLETE 0
-
-simple conn=joe,user=joe
-EXPLAIN SELECT * FROM v;
-----
-db error: ERROR: permission denied for SCHEMA "materialize.public"
+EOF
+COMPLETE 1
 
-simple conn=child,user=child
+simple multiline,conn=child,user=child
 EXPLAIN SELECT * FROM v;
 ----
-db error: ERROR: permission denied for SCHEMA "materialize.public"
-
-simple conn=mz_system,user=mz_system
-GRANT USAGE ON SCHEMA materialize.public TO view_owner;
-----
-COMPLETE 0
+Explained Query:
+  ReadStorage materialize.public.t
 
-simple conn=joe,user=joe
-EXPLAIN SELECT * FROM v;
-----
-db error: ERROR: permission denied for TABLE "materialize.public.t"
+EOF
+COMPLETE 1
 
-simple conn=child,user=child
+simple multiline,conn=child,user=child
 EXPLAIN SELECT * FROM v;
 ----
-db error: ERROR: permission denied for TABLE "materialize.public.t"
-
-simple conn=mz_system,user=mz_system
-GRANT SELECT ON TABLE t TO view_owner;
-----
-COMPLETE 0
-
-simple conn=mz_system,user=mz_system
-REVOKE SELECT ON TABLE t FROM view_owner;
-----
-COMPLETE 0
-
-simple conn=mz_system,user=mz_system
-REVOKE SELECT ON TABLE v FROM joe;
-----
-COMPLETE 0
+Explained Query:
+  ReadStorage materialize.public.t
 
-simple conn=mz_system,user=mz_system
-REVOKE USAGE ON SCHEMA materialize.public FROM joe;
-----
-COMPLETE 0
+EOF
+COMPLETE 1
```

Comment:
It looks like these two test cases are identical? I think we can remove one.

Reply:
They target different connection and user parameters:

```
simple conn=joe,user=joe
EXPLAIN SELECT * FROM v;
----
db error: ERROR: permission denied for SCHEMA "materialize.public"

simple conn=child,user=child
EXPLAIN SELECT * FROM v;
----
db error: ERROR: permission denied for SCHEMA "materialize.public"
```